gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/BuildController.groovy

```diff
@@ -25,6 +26,7 @@ import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestMethod
 import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.RequestParam
```

Comment:
Looks like the alphabetical sort order for this group is off here.  `RequestParam` comes before `RestController`

Reply:
You are right. I didn't run formatter on existing code to avoid unnecessary diff. Fixed.